guava/hw10-workHibernate/src/main/java/ru/otus/DbServiceDemo.java

```diff
@@ -0,0 +1,47 @@
+package ru.otus;
+
+import org.hibernate.SessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.otus.core.dao.UserDao;
+import ru.otus.core.model.AddressDataSet;
+import ru.otus.core.model.PhoneDataSet;
+import ru.otus.hibernate.dao.UserDaoHibernate;
+import ru.otus.core.service.DBServiceUser;
+import ru.otus.core.service.DbServiceUserImpl;
+import ru.otus.core.model.User;
+import ru.otus.hibernate.HibernateUtils;
+import ru.otus.hibernate.sessionmanager.SessionManagerHibernate;
+
+import java.util.*;
+
+public class DbServiceDemo {
+    private static Logger logger = LoggerFactory.getLogger(DbServiceDemo.class);
+
+    public static void main(String[] args) {
+        // Все главное см в тестах
+        SessionFactory sessionFactory = HibernateUtils.buildSessionFactory("hibernate.cfg.xml", User.class);
```

Comment:
```suggestion
        SessionFactory sessionFactory = HibernateUtils.buildSessionFactory(
                "hibernate.cfg.xml",
                User.class,
                AddressDataSet.class,
                PhoneDataSet.class);
```

Забыли про другие классы